Fix typos on S3 sink documentation (#4862)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sinks/s3.md

The `parquet` codec writes events into a Parquet file.
You must set the `buffer_type` to `multipart` when using Parquet.

> The Parquet codec writes data using the Avro schema. In general, you should define your own schema because it will most accurately reflect your needs.

Reviewer: ```suggestion
The Parquet codec writes data using the Avro schema. However, we generally recommend that you define your own schema so that it can best meet your needs.
```
Just an option for wording.
